README.md

```diff
@@ -1,88 +1,112 @@
 
 ## API
 
-Though we are usually pretty good at backward compatibility, there is no guarantee.
-This is an unpaid project. Bnd properties will be backward compatible if there is any possibility.
+We go out of our way to be backward compatible. There are thousands of extensive test cases and we use our own [baselining tool][1] to verify that we do not make gratuitious changes. Since this tool is part of Bndtools we often find out early enough that there are alternatives to breaking changes. We strictly follow OSGi semantic versioning. This means that we mark exported packages with a package version.
 
-If you're building a tool with a general audience  that includes Bnd for a general audience I would appreciate if you got in touch with us so we can keep in touch. We are always interested in ideas.
+We try to keep major versions to a minimum but sometimes we must. This is clearly marked with a major release number. 
 
-## Eclipse
+The master branch is release 7 currently and the Java release version is 17. Since some libraries are used in the embedded world that have a slower uptake of the Java release train we keep a classic branch that continues a major release 6 on Java 1.8. The classic branch does not include the plugins.
 
-Bndtools is the development environment of Bnd.
+## Contributing
 
-## Building
+Want to work on Bnd/Bndtools? There are [instructions](CONTRIBUTING.md) to get you started. Please let us know if anything feels wrong or incomplete.
 
-We use Gradle and Maven to build and the repo includes `gradlew` and `mvnw` at the necessary versions.
-We require at least Java 17.
+## Repo Structure
 
-- `./gradlew :build` - Assembles and tests the Bnd Workspace projects. This must be run before building the Bnd Maven and Gradle plugins.
-- `./gradlew :gradle-plugins:build` - Assembles and tests the Bnd Gradle plugins.
-- `./mvnw install` - Assembles and tests the Bnd Maven plugins.
-- `./gradlew :publish` - Assembles and publishes the Bnd Workspace projects into `dist/bundles`.
-- `./gradlew :gradle-plugins:publish` - Assembles and publishes the Bnd Gradle plugins into `dist/bundles`.
-- `./mvnw -Pdist deploy` - Assembles and publishes the Bnd Maven plugins into `dist/bundles`.
+### Branches 
 
-Bnd/Bndtools is continuously built on [GitHub Actions](https://github.com/bndtools/bnd/actions/workflows/cibuild.yml).
-[![GitHub Actions CI Build Status](https://github.com/bndtools/bnd/actions/workflows/cibuild.yml/badge.svg)](https://github.com/bndtools/bnd/actions/workflows/cibuild.yml)
+We have the following [branches](https://github.com/bndtools/bnd/branches/all):
 
-[CodeQL](https://github.com/bndtools/bnd/security/code-scanning?query=tool%3ACodeQL) is used for continuous security analysis.
+* `master` – Where the work for the next release takes place. Pushes on this branch release to the snapshot repository
+* `classic` – A continuation of 6.4 compiled on Java 1.8 for the poor developers stuck in the limbo of 1.8 and 17.
```

Comment:
How are you maintaining the dependencies on the classic branch? Do you keep them all updated like master? What version of bnd is used to build classic? The latest bnd snapshot from master? There should be some description of how classic is kept up to date.

Reply:
I've added a section to the wiki and will update it after the first release of classic.